### PR TITLE
CP-18010: Fix typo of setting xapi.conf

### PR DESCRIPTION
**Pull request by cheng-z**
Signed-off-by: Cheng Zhang <cheng.zhang@citrix.com>

### CODE REVIEW

**Comment by cheng-z** [comment]
Hi @robhoes 
Please help to merge this typo in xapi_globs.ml.

File: ocaml/xapi/create_misc.ml [hunk 1]
```diff
@@ -624,10 +624,10 @@ let create_chipset_info ~__context =
 	Db.Host.set_chipset_info ~__context ~self:host ~value:info
 
 let create_patches_requiring_reboot_info ~__context ~host =
-  let patch_uuids = try Stdext.Unixext.read_lines !Xapi_globs.reboot_required_hfxs with _ -> [] in
+  let patch_uuids = try List.sort_uniq String.compare (Stdext.Unixext.read_lines !Xapi_globs.reboot_required_hfxs) with _ -> [] in
```
Comment (robhoes) [inline ocaml/xapi/create_misc.ml]: We normally use `List.setify` for this (we don't care about sorting here).

File: ocaml/xapi/xapi_globs.ml [hunk 2]
```diff
@@ -1017,10 +1017,10 @@ let other_options = [
   "reboot_required_hfxs", Arg.Set_string reboot_required_hfxs,
     (fun () -> !reboot_required_hfxs), "File to query hotfix uuids which require reboot";
 
-  "xen_livepatch_list", Arg.Set_string reboot_required_hfxs,
+  "xen_livepatch_list", Arg.Set_string xen_livepatch_list,
```
Comment (robhoes) [inline ocaml/xapi/xapi_globs.ml]: Good catch... I missed that in the previous PR!

**Comment by robhoes** [comment]
:shipit: (just waiting for Travis).

**Pull request merged**